specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/examples/createSubscription.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "scope": "/providers/Microsoft.Billing/billingAccounts/a460e639-9c9b-4b58-9046-1120b22c7dc1/",
```

Comment:
Please update the examples to reflect your latest change.

Reply:
done